modules/core/test/v2/unit/coins/sol.ts

```diff
@@ -75,7 +93,26 @@ describe('SOL:', function () {
 
   describe('Sign transaction:', () => {
     it('should sign transaction', async function () {
-      await should.throws(() => basecoin.signTransaction('placeholder'), 'signTransaction method not implemented');
+      const signed = await basecoin.signTransaction({
+        txPrebuild: {
+          txBase64: resources.RAW_TX_UNSIGNED,
+          keys: [resources.accountWithSeed.publicKey.toString()],
+          addressVersion: 1,
```

Comment:
what's `addressVersion` used for? is it required?
if not, let's remove it.

Reply:
removed